neo4j/datadog_checks/neo4j/data/conf.yaml.example

```diff
@@ -2,341 +2,45 @@
 #
 init_config:
 
-    ## @param proxy - mapping - optional
-    ## Set HTTP or HTTPS proxies for all instances. Use the `no_proxy` list
-    ## to specify hosts that must bypass proxies.
-    ##
-    ## The SOCKS protocol is also supported like so:
-    ##
-    ##   socks5://user:pass@host:port
-    ##
-    ## Using the scheme `socks5` causes the DNS resolution to happen on the
-    ## client, rather than on the proxy server. This is in line with `curl`,
-    ## which uses the scheme to decide whether to do the DNS resolution on
-    ## the client or proxy. If you want to resolve the domains on the proxy
-    ## server, use `socks5h` as the scheme.
-    #
-    # proxy:
-    #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
-    #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
-    #   no_proxy:
-    #   - <HOSTNAME_1>
-    #   - <HOSTNAME_2>
-
-    ## @param skip_proxy - boolean - optional - default: false
-    ## If set to `true`, this makes the check bypass any proxy
-    ## settings enabled and attempt to reach services directly.
-    #
-    # skip_proxy: false
-
-    ## @param timeout - number - optional - default: 10
-    ## The timeout for connecting to services.
-    #
-    # timeout: 10
-
-    ## @param service - string - optional
-    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
-    ##
-    ## Additionally, this sets the default `service` for every log source.
-    #
-    # service: <SERVICE>
-
 ## Every instance is scheduled independent of the others.
 #
 instances:
 
-    ## @param neo4j_url - string - required
-    ## Neo4J url to connect to.
+    ## @param host - string - required
+    ## Neo4j host to connect to
     #
-  - neo4j_url: localhost
+  - host: localhost
```

Comment:
You're going to break backwards compatibility for users already running the neo4j integration. Nothing blocking, just want to make sure you know about this

Reply:
You can avoid this by reading both options in the code

Reply:
We made a lot of changes in 4.x so wanted a clean integration.
This code base will work with 3.5.